servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -43,6 +43,7 @@
   private boolean mIsComplete = false;
   private boolean mCache = false;
   private String mUfsPath = "";
+  private long mTTL;
```

Comment:
Hrmmm, ok. Is Tachyon going to support this api later? It looks like it just sets the TTL when created.